clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -84,7 +83,6 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
                   Instant.ofEpochSecond(
                       auditLogRequest.getTimestamp().getSeconds(),
                       auditLogRequest.getTimestamp().getNanos()))
-              .setResource(MonitoredResource.newBuilder(MONITORED_RESOURCE_TYPE).build())
```

Comment:
This change is intended. I found that our go code https://github.com/abcxyz/lumberjack/blob/554bab0378f22d2935cb8630e4c7e89f3fc8a908/clients/go/pkg/cloudlogging/processor.go#L128 doesn't override the `resource` field. I think we need to make both java and go consistent in setting the `resource` field. And based on our doc, this field can be auto generated by cloud logging. I don't see any value of overridden it by ourselves.

Reply:
In addition, this part causes null fields in json maps which triggers incomplete unmarshal due to limitation in protojson. With the limitation of https://github.com/golang/protobuf/issues/1313, we won't be able to verify the timestamp is set in our e2e test and our test is also fragile(fields whose order is after `resource` field can't be unmarshalled). Therefore, I try to make the test logs without any null value in json maps or json arrays. I still prefer not to changing the src code just to make test log consistent. But in this case, we have inconsistency in java and go src code when setting `resource` field, I think it makes sense to let java follow the go business logic which let cloud logging to auto generate the `resource` field.

Reply:
No. I thought cloud logging would fill this part. It's not critical to the audit logging.